test/extended/networking/tuning.go

```diff
@@ -43,6 +44,53 @@ var whitelistedSysctls = []SysctlVariant{
 	// {Sysctl: "net.ipv6.neigh.IFNAME.retrans_time_ms", Value: "1010", Path: "/proc/sys/net/ipv6/neigh/net1/retrans_time_ms"},
 }
 
+// getLowestIfindexInterfaceFromUeventData parses the output of one or more concatenated 'uevent' files
+// from /sys/class/net/*/uevent and returns the name of the interface with the lowest ifindex
+func getLowestIfindexInterfaceFromUeventData(data string) (string, error) {
+	const interfaceTag string = "INTERFACE="
+	const ifindexTag string = "IFINDEX="
+
+	// Results will include at least INTERFACE and IFINDEX items for each interface.
+	// Ignore optional DEVTYPE.
+	sanitized := make([]string, 0)
+	for _, line := range strings.Fields(data) {
+		if strings.HasPrefix(line, interfaceTag) || strings.HasPrefix(line, ifindexTag) {
+			sanitized = append(sanitized, line)
+		}
+	}
+	if len(sanitized)%2 != 0 {
+		return "", fmt.Errorf("bad sanitized uevent data length; expected multiple of two got %d", len(sanitized))
```

Comment:
make sense to also output the full "data" here to help with debugging whatever problem lands us here?

Reply:
Fixed.